services/servo.c

```diff
@@ -43,12 +44,18 @@ static void set_pwr(srv_t *state, int on) {
     if (state->is_on == on)
         return;
     if (on) {
+        if (state->power_pin != 0xff) {
+            pin_setup_output(state->power_pin);
+            pin_set(state->power_pin,0);
+        }
```

Comment:
no need to guard - the pin_* functions don't do anything when `pin==0xff`

Reply:
Ok! Will fixup